leo_bringup/scripts/leo_system

```diff
@@ -17,11 +17,19 @@ def shutdownCallback(x):
     os.system("shutdown -h now")
 
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
```

Comment:
I have a feeling `robot_namespace` would be more appropriate name for the topic. This also applies to LeoRover/leo_ui#6